Task4/src/main/java/by/innowise/tasks/entity/Speaker.java

```diff
@@ -0,0 +1,37 @@
+package by.innowise.tasks.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "speaker")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@EqualsAndHashCode
```

Comment:
@ Data annotation already provide equals and hashCode methods. You can remove this annotation.